lib/command/src/Obelisk/Command/Project.hs

```diff
@@ -61,7 +61,7 @@ import System.IO.Temp
 import System.IO.Unsafe (unsafePerformIO)
 import System.PosixCompat.Files
 import System.PosixCompat.Types
-import System.PosixCompat.User
```

Comment:
I doubt we want to go out of our way to keep using an old version of `unix-compat`, for the sake of a single function which simply crashes: https://github.com/jacobstanley/unix-compat/pull/62/files#diff-93ed634b109420481984ea0fecc52a958c1d98c2728f8ec0b4c6ed741a8d4a24L55-L63

If `ob` survives long enough for windows support to be a concern, a lot of things will need to be revisited anyway.